src/main.py

```diff
@@ -204,6 +273,45 @@ def do_recognition(args, recorder, recognizer, player):
             time.sleep(1)
 
 
+class StatusUi(object):
+
```

Comment:
The blank-line-before-docstring style is used consistently at the project. However, it seems it's based on an old version of PEP8: https://github.com/hhatto/autopep8/issues/194

I'd prefer to fix in a another PR.